cmd/open.js

```diff
@@ -10,25 +10,16 @@ exports.builder = (yargs) =>
     yargs
     .strict()
     .usage('Usage: $0 open <package> [options]\n\nOpens the package in your browser..')
-    .example('$0 open gulp', 'Opens "gulp" package')
-    .example('$0 open --npms gulp', 'Opens "gulp" package in `https://npms.io` service')
-    .example('$0 open --npm gulp', 'Opens "gulp" package in `https://npmjs.org` service')
-    .example('$0 open --github gulp', 'Opens "gulp" package in `https://github.com` service')
+    .example('$0 open gulp', 'Opens "gulp" package using auto source')
+    .example('$0 open --link npms gulp', 'Opens "gulp" package in `https://npms.io` service')
+    .example('$0 open --link npm gulp', 'Opens "gulp" package in `https://npmjs.org` service')
+    .example('$0 open --link auto gulp', 'Opens "gulp" package in `https://github.com` service')
     .demand(1, 1)
     .options({
```

Comment:
See comment below, it will fall through if `--link npm` or `--link npms` not supplied.